pkg/harvester-manager/l10n/en-us.yaml

```diff
@@ -7,6 +7,19 @@ harvesterManager:
     description: Harvester is a modern Hyperconverged infrastructure (HCI) solution built for bare metal servers using enterprise-grade open source technologies including Kubernetes, Kubevirt and Longhorn.
   plugins:
     loadError: Error loading harvester plugin
+  extension:
+    install:
+      warning: "Harvester extension is not installed"
+      steps:
+        repo:
+          1: Go to
+          2: Manage Repositories
+          3: and add the Harvester repository
+        ui:
+          1: Go to
+          2: Extensions
+          3: page and install the Harvester Extension
```

Comment:
I would suggest to add hyperlink https://github.com/harvester/harvester-ui-extension.git on the word `Harvester Extension` or directly show https://github.com/harvester/harvester-ui-extension.git to user.

As a rancher user, I don't know what's harvester extension and where is its github repository.